content/en/blog/_posts/2022-11-21-devicemanager-ga.md/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: blog
+title: 'Kubernetes 1.26: Device Manager graduates to GA'
+date: 2022-12-19
+slug: devicemanager-ga
+---
+
+**Author:** Swati Sehgal (Red Hat)
+
+The Device Plugin framework was introduced in Kubernetes v1.8 release as a vendor
+independent framework to enable discovery, advertisement and allocation of external
+devices without modifying core Kubernetes. The feature graduated to Beta in v1.10.
+With the recent release of Kubernetes v1.26, Device Manager is now generally
+available (GA).
+
+Device Manager in kubelet facilitates communication with device plugins over gRPC
+through Unix sockets. Device Manager and Device plugins both act as gRPC servers
+and clients by serving and connecting to the exposed gRPC services respectively.
+Device plugins serve a gRPC service that kubelet connects to for device discovery,
+advertisement (as extended resources) and allocation. Device Manager connects to
+the `Registration` gRPC service served by kubelet to register itself with kubelet.
+
+Please refer to the documentation for an [example](/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/#example-pod) on how a pod can request a device exposed to the cluster by a device plugin.
+
+Here are some device plugins for example implmentations:
+- [NVIDIA device plugin for Kubernetes](https://github.com/NVIDIA/k8s-device-plugin)
+- [Collection of Intel device plugins for Kubernetes](https://github.com/intel/intel-device-plugins-for-kubernetes)
+- [SRIOV network device plugin for Kubernetes](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin)
+- [AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
```

Comment:
Alphabetical order, please:
```suggestion
- [AMD GPU device plugin](https://github.com/RadeonOpenCompute/k8s-device-plugin)
- Collection of [Intel device plugins for Kubernetes](https://github.com/intel/intel-device-plugins-for-kubernetes)
- [NVIDIA device plugin for Kubernetes](https://github.com/NVIDIA/k8s-device-plugin)
- [SRIOV network device plugin for Kubernetes](https://github.com/k8snetworkplumbingwg/sriov-network-device-plugin)
```

Another option is a fair random ordering. Alphabetical order is easier though!

Reply:
I had ordered them by number of stars on github previously, changed to alphabetical order. Thanks!